app/helpers/bulk_downloads_helper.rb

```diff
@@ -0,0 +1,27 @@
+module BulkDownloadsHelper
+  SAMPLE_NO_PERMISSION_ERROR = "You do not have permission to access all of the selected samples. Please contact us for help.".freeze
```

Comment:
Why would they even see it in this case?
If this a potential impossible state, maybe we should be alerted for it.

Reply:
Great point about alerting. I added an alert to the controller create method.